src/app/controller/taskLgtm.ts

```
@@ -0,0 +1,10 @@
+import { logInfo } from "@/Logger";
+
+/**
+ * タスクLGTMを実行する
+ * @param {any} payload - ペイロード
+ * @param {string} task_id - タスクID
+ */
+export function taskLgtm(payload: any): void {
+  logInfo("taskLgtm");
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)